packages/ckeditor5-coremedia-link/src/linktarget/config/LinkTargetDefaultRuleDefinition.ts

```diff
@@ -0,0 +1,68 @@
+export type TargetDefaultRuleDefinition = TargetDefaultRuleDefinitionWithFilter | TargetDefaultRuleDefinitionWithType;
+
+/**
+ *
+ */
+export interface TargetDefaultRuleDefinitionWithFilter {
+  /**
```

Comment:
Maybe move a part of the comment to the interface documentation. First part feels like the interface documentation, last part like the function documentation.